.github/mcp/mcp_pytest.py

```diff
@@ -26,20 +26,24 @@
     parser.add_argument('--timeout', type=int, default=1800, help='Timeout for run (in seconds)')
     args = parser.parse_args()
 
+    name = args.name
     git_integration = {
         'integration_type': 'git_repo',
         'git_repo': 'mosaicml/composer',
         'ssh_clone': 'False',
     }
     if args.git_branch is not None and args.git_commit is None:
+        name += f'-branch-{args.git_branch}'
```

Comment:
Do we need to do any re-formatting here so that the branch string is formatted correctly?  For example I don't think a branch with '/' in the name will work in the run name.

Reply:
MCLOUD should reformat for us:
```
Invalid run name "sd/baseline-112": Run names must be less than 63 characters and contain only lower-case letters, numbers, or "-". Converting to a valid name: sd-baseline-112
```
I've added a cap for max length